types/dagre-d3/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/cpettitt/dagre-d3
 // Definitions by: Mark Wong Siang Kai <https://github.com/markwongsk>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```

Comment:
this depends on d3, which you just updated to `2.3` so should not this be `2.3` as well?